apps/graphql/src/dataloaders/Itinerariesloader.js

```diff
@@ -48,11 +84,10 @@ const parseDate = (date: Date) =>
   DateTime.fromJSDate(date, { zone: 'UTC' }).toFormat(dateFormat);
 
 export const parseParameters = (input: ItinerariesSearchParameters) => {
-  const dateTo = input.dateTo ?? input.dateFrom;
   const params = {
     flyFrom: input.travelFrom,
     dateFrom: parseDate(input.dateFrom),
-    dateTo: parseDate(dateTo),
+    dateTo: parseDate(input.dateFrom),
```

Comment:
Why is dateTo pointing to input.dateFrom?

Reply:
Because API is prepared for date intervals, not for specific date as we want right now. So we have to use `dateFrom` and `dateTo` as a departure and `returnFrom` and `returnTo` as an arrival.